frontend/views/containers/group-settings/GroupRulesSettings.vue

```diff
@@ -0,0 +1,160 @@
+<template lang='pug'>
+  ul.c-wrapper
+    li.cardBox(
+      v-for='rule in votingRulesSorted'
+      :class='{ isActive: isRuleActive(rule) }'
+      :aria-current='isRuleActive(rule)'
+    )
+      p.is-title-4 {{ config[rule].title }}
+      p.has-text-1.c-expl(v-html='config[rule].explanation')
+      i18n.pill.is-primary.c-active(v-if='isRuleActive(rule)') Active
+
+      dl.c-status(v-if='isRuleActive(rule)')
+        dt.c-status-term {{ config[rule].status }}
+        dd.c-status-desc
+          span(v-html='votingValue(rule)')
+          i18n.link(
+            tag='button'
+            @click='openVotingProposal(rule)'
+          ) Change
+
+      banner-simple.c-banner(
+        severity='info'
+        v-if='isVotingRuleAdjusted(rule)'
+      )
+        i18n * This value was automatically adjusted because your group is currently smaller than the disagreement number.
+
+      i18n.link(
+        v-if='!isRuleActive(rule)'
+        tag='button'
+        @click='openVotingProposal(rule)'
+      ) Change to this voting system
+</template>
+
+<script>
+import sbp from '~/shared/sbp.js'
+import { mapGetters } from 'vuex'
+import { RULE_PERCENTAGE, RULE_DISAGREEMENT } from '@model/contracts/voting/rules.js'
+import { OPEN_MODAL } from '@utils/events.js'
+import L from '@view-utils/translations.js'
+import BannerSimple from '@components/banners/BannerSimple.vue'
+
+export default {
+  name: 'GroupVotingSystem',
```

Comment:
Note: This file was not created, it was renamed: GroupVotingSystem -> GroupRulesSettings.

And added the needed logic to connect to the state. The markup is pretty much the same and the CSS wasn't modified at all.